apps/codeforafrica/src/components/SocialMediaConnectButton/SocialMediaConnectButton.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env browser */
+import SvgIcon from "@mui/material/SvgIcon";
+import React from "react";
+
+import FacebookIcon from "@/codeforafrica/assets/icons/Type=facebook, Size=24, Color=CurrentColor.svg";
+import GithubIcon from "@/codeforafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import InstagramIcon from "@/codeforafrica/assets/icons/Type=instagram, Size=24, Color=CurrentColor.svg";
+import LinkedinIcon from "@/codeforafrica/assets/icons/Type=linkedin, Size=24, Color=CurrentColor.svg";
+import SlackIcon from "@/codeforafrica/assets/icons/Type=slack, Size=24, Color=CurrentColor.svg";
+import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
+import SocialMediaButton from "@/codeforafrica/components/SocialMediaButton";
+
+function getButtonIcon(name) {
+  switch (name) {
+    case "facebook":
+      return <SvgIcon sx={{ fill: "none" }} component={FacebookIcon} />;
```

Comment:
```suggestion
  switch (name) {
    case "facebook":
      return FacebookIcon;
```

Just return the `Icon` and do the rendering in main component.